src/main/java/com/minecolonies/compatibility/tinkers/SlimeTreeCheck.java

```diff
@@ -77,10 +79,8 @@ public boolean checkForTinkersSlimeLeaves(@NotNull final Block block)
      * @param block the block.
      * @return if the block is a slime sapling.
      */
-    @Override
-    @Optional.Method(modid = "tconstruct")
-    public boolean checkForTinkersSlimeSapling(@NotNull final Block block)
+    public static boolean isSlimeSapling(@NotNull final Block block)
     {
-        return block == TinkerWorld.slimeSapling;
+        return new SlimeTreeCheck().checkForTinkersSlimeSapling(block);
```

Comment:
Isn't it easier if we do that with static methods and a static method call without creating an instance?